src/markdown-pages/build-apps/serve-publish-subscribe/catalog.mdx

```diff
@@ -156,7 +169,7 @@ Update your _config.json_ file:
 
 <Step>
 
-Include screenshots in your _screenshots_ directory.
+Include screenshots in your root _screenshots_ directory or any artifact _screenshots_ directory.
```

Comment:
Same thought here about "artifact" vs "Nerdpack item".

Reply:
Fixed in 4edaa7f